### PR TITLE
Replace Gnosis Protocol --> CoW Protocol

### LINKED ISSUE #2119: [1.8.1] 'Gnosis protocol' is not changes to CoW Protocol for 'Unsupported token' message
1. Open Swap form
2. Connect to Gnosis chain
3. Select token pair: WXDAI-ADAI
4. Press on the 'Read more about...'  link

**AR**:  'Gnosis protocol'  is mentioned in the message. Also, it is displayed in tooltip for an unsupported token in the token list

**ER**: 'CoW Protocol' should be

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Fixes #2119
image
image

  # To Test

1. Open Swap form
2. Connect to Gnosis chain
3. Select token pair: WXDAI-ADAI
4. Enter an amount.
5. Press on the 'Read more about...' link

### CODE REVIEW

File: src/custom/api/gnosisProtocol/api.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ const DEFAULT_HEADERS = {
   'Content-Type': 'application/json',
   'X-AppId': APP_DATA_HASH.toString(),
 }
-const API_NAME = 'Gnosis Protocol'
+const API_NAME = 'CoW Protocol'
```
Comment (alongoni) [inline src/custom/api/gnosisProtocol/api.ts:76]: not sure about this change, maybe it can cause any problem?

**Review by W3stside — APPROVED** [review]
should be fine, thanks!

**Review by elena-zh — APPROVED** [review]
Changes LGTM!

File: src/custom/api/gnosisProtocol/api.ts (hunk 1 above)
Comment (henrypalacios) [reply src/custom/api/gnosisProtocol/api.ts:76]: No problem, it is used for messages.

**Review by henrypalacios — APPROVED** [review]
Great Job!

**Review by anxolin — COMMENTED** [review]
APPROVE

**Pull request merged**